torch/nn/modules/rnn.py

```diff
@@ -518,49 +515,50 @@ class LSTM(RNNBase):
         bias: If ``False``, then the layer does not use bias weights `b_ih` and `b_hh`.
             Default: ``True``
         batch_first: If ``True``, then the input and output tensors are provided
-            as (batch, seq, feature). Default: ``False``
+            as `(batch, seq, feature)` instead of `(seq, batch, feature)`.
+            Note that this does not apply to hidden or cell states. See the
+            Inputs/Outputs sections below for details.  Default: ``False``
         dropout: If non-zero, introduces a `Dropout` layer on the outputs of each
             LSTM layer except the last layer, with dropout probability equal to
             :attr:`dropout`. Default: 0
         bidirectional: If ``True``, becomes a bidirectional LSTM. Default: ``False``
         proj_size: If ``> 0``, will use LSTM with projections of corresponding size. Default: 0
 
-    Inputs: input, (h_0, c_0)
-        - **input** of shape `(seq_len, batch, input_size)`: tensor containing the features
-          of the input sequence.
-          The input can also be a packed variable length sequence.
+    Inputs: `input, (h_0, c_0)`
```

Comment:
nit: The font makes this look like h_o... but I don't think we should/can do anything about that here unless you see a way to fix it.

Reply:
I noticed that too :( Looks like removing the backticks helps so I'll do that